components/event-bus/internal/knative/util/eventing.go

```diff
@@ -85,6 +86,9 @@ func (k *KnativeLib) GetChannel(name string, namespace string) (*evapisv1alpha1.
 		log.Printf("ERROR: GetChannel(): geting channel: %v", err)
 		return nil, err
 	} else {
```

Comment:
If already has a `return`, so you don't need to use `else` here.

Reply:
the "channel" is bound to the "if" context